.env

```diff
@@ -0,0 +1 @@
+DB_URL=postgres://vapor_username:vapor_password@localhost:5432/vapor_database
```

Comment:
Not sure if `.env` should be committed
+ previous version was using sqlite not postgres